### DISCUSSION TITLE
GUI parallel plotting

### DISCUSSION THREAD

**Discussion by dorkmo** [Chia-Network/chia-blockchain#677] [Q&A] [answered]
As noted in the release notes, you currently cant queue up x number of 2 plottings. But a new feature is to set a delay before starting. Does it work to ladder a bunch of start times?

In the past id set it to plot the max number, and do again in a couple days. Whats the best practice now?

**Comment by Crypt0fun22** [comment]
I'm am interested as well to the question that dorkmo is asking.

**Reply by Crypt0fun22** [reply]
Is there a "how to" for parallel plotting. It would be nice to have Q&A for the individual settings in plotting section of the UI. If anyone has info to contribute I would like create a resource for everyone and new users

**Comment by hoffmang9** [comment, marked as answer]
Support for this was included in Beta 22 yesterday - https://github.com/Chia-Network/chia-blockchain/releases/tag/1.0beta22
